packages/@webex/internal-plugin-calendar/src/calendar.js

```diff
@@ -312,6 +349,118 @@ const Calendar = WebexPlugin.extend({
         return Promise.all(promises).then(() => meetingObjects);
       });
   },
+
+  /**
+   * Create calendar event
+   * @param {object} [data] meeting payload data
+   * @returns {Promise} Resolves with creating calendar event response
+   * */
+  createCalendarEvent(data) {
+    return EncryptHelper.encryptCalendarEventRequest(this, data).then(() => {
```

Comment:
can we use wait will look little cleaner

Reply:
unnecessary `return` statement, should be able to `.then(() => this.request({ /* ... */ });`

Reply:
@InteractiveTimmy done